File: Java/tripletexting.java
/*
 *	Problem: Triple Texting
 */

import java.util.Scanner;

public class tripletexting {
	
	public static void produce_word(String input) {
		int wordLength = input.length() / 3;
		
		String a = input.substring(0,wordLength);
		String b = input.substring(wordLength,wordLength*2);
		String c = input.substring(wordLength*2);
		
		if ( a.equals(b) ) System.out.println(a);
		else if ( a.equals(c) )	System.out.println(a);
		else System.out.println(b);
	} // end produce_word
	
	/*-----main-----*/
	public static void main(String[] args) {		
		Scanner sc = new Scanner(System.in);
		String input = sc.next();
		produce_word(input);
	} // end main
	
}

File: Python/tripletexting.py
'''
 *
 *	Problem: "Triple Texting"
 *
'''

#-----produce_word-----#
def produce_word(line):
	wordLength = len(line) / 3
	wordLength = round(wordLength)
	
	a = line[:wordLength]
	b = line[wordLength:(wordLength*2)]
	c = line[(wordLength*2):]
	
	if (a == b): return a
	elif (a == c): return a
	else: return b
	
#-----main-----#
line = input()
output = produce_word(line)
print(output)
